app/helpers/application_helper.rb

```diff
@@ -91,6 +91,14 @@ def contacts_at_your_instutution_path
     end
   end
 
+  def sso_english_guidlines_path  
+    'https://drive.google.com/file/d/11m2lepJF7207uwqKwR0TzWDcPAZyq6Kb/view?usp=drive_link'
+  end
+
+  def sso_french_guidlines_path
+    'https://drive.google.com/file/d/1QIVhN6P566xUaOyVZMwtfz8a7fI4X1D_/view?usp=drive_link'
+  end
```

Comment:
Since we are showing both links instead of changing them with different locales we should add them in the view instead of having these methods. What do you think ?